csharp/src/Apache.Arrow.Flight.Sql/FlightExtensions.cs

```diff
@@ -0,0 +1,81 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+
+namespace Apache.Arrow.Flight.Sql;
+
+internal static class FlightExtensions
+{
+    public static byte[] PackAndSerialize(this IMessage command) => Any.Pack(command).ToByteArray();
+    public static T ParseAndUnpack<T>(this ByteString source) where T : IMessage<T>, new() => Any.Parser.ParseFrom(source).Unpack<T>();
+
+    public static IEnumerable<long> ExtractRowCount(this RecordBatch batch)
```

Comment:
I have to admit, I can't tell what the semantics of this are supposed to be :(. I did a quick search to see what the result of `ExecuteUpdate` was supposed to look like but that didn't help. Putting the explanation in a comment might be worthwhile.

Reply:
Added:
Executes an update SQL command and returns the number of affected rows.

But after looking at:
```c++
ARROW_ASSIGN_OR_RAISE(auto updated_rows,
                          sql_client->ExecuteUpdate({}, "UPDATE STATEMENT"));
```

I created the same method and implemented it as is.